src/machines/java/com/enderio/machines/common/blockentity/XPVacuumBlockEntity.java

```diff
@@ -55,9 +55,9 @@ public void handleEntity(ExperienceOrb xpe) {
     @Override
     protected @Nullable FluidTank createFluidTank() {
         return new FluidTank(Integer.MAX_VALUE) {
+
             @Override
             protected void onContentsChanged() {
-                super.onContentsChanged();
```

Comment:
Why did we remove the call to super here? I'd rather we keep these, because if we moved FluidTank to another implementation which implemented this, we would be skipping it's implementation and possibly unknowingly breaking things.